README.md

```diff
@@ -1 +1,188 @@
-# cloudant-spring
+# Cloudant Spring Library
+
+The is a library to provide Spring developers easy configuration of the [official Cloudant library for Java](https://github.com/cloudant/java-cloudant/).
+
+The library is split into two parts:
+* `cloudant-spring-boot-starter` for [Spring Boot](https://projects.spring.io/spring-boot/) applications
+
+* `cloudant-spring-framework` for [Spring Framework](https://projects.spring.io/spring-framework/) applications
+
+
+* [Installation and Usage](#installation-and-usage)
+* [Getting Started](#getting-started)
+* [Related Documentation](#related-documentation)
+* [Development](#development)
+    * [Contributing](CONTRIBUTING.md)
+    * [Test Suite](CONTRIBUTING.md#running-the-tests)
+    * [Using in Other Projects](#using-in-other-projects)
+    * [License](#license)
+    * [Issues](#issues)
+
+## Installation and Usage
+
+### Spring Boot Applications
+
+Gradle:
+```groovy
+dependencies {
+    compile group: 'com.cloudant', name: 'cloudant-spring-boot-starter', version: '0.0.1'
+}
+```
+
+Maven:
+~~~ xml
+<dependency>
+  <groupId>com.cloudant</groupId>
+  <artifactId>cloudant-spring-boot-starter</artifactId>
+  <version>0.0.1</version>
+</dependency>
+~~~
+
+### Spring Framework Applications
+
+Gradle:
+```groovy
+dependencies {
+    compile group: 'com.cloudant', name: 'cloudant-spring-framework', version: '0.0.1'
+}
+```
+
+Maven:
+~~~ xml
+<dependency>
+  <groupId>com.cloudant</groupId>
+  <artifactId>cloudant-spring-framework</artifactId>
+  <version>0.0.1</version>
+</dependency>
+~~~
+
+## Getting Started
+
+This section contains simple examples of connecting to Cloudant using the two libraries.
+
+### Spring Boot Applications
+
+You must provide the following properties:
+
+* cloudant.url
+* cloudant.username
+* cloudant.password
+
+For example in an `application.properties` file:
+
+~~~
+cloudant.url=http://cloudant.com
+cloudant.username=myUsername
+cloudant.password=myPassword
+~~~
+
+Spring Boot will create a `com.cloudant.client.api.CloudantClient` bean that can be used to interact with your Cloudant instance:
+
+~~~ java
+@Autowired
+private CloudantClient client;
+
+public List<String> getAllDbs() {
+    return client.getAllDbs();
+}
+~~~
+
+The Spring Boot starter will also provide a `com.cloudant.client.api.Database` bean if your provide a `cloudant.db` property. For example:
+
+~~~
+cloudant.url=http://cloudant.com
+cloudant.username=myUsername
+cloudant.password=myPassword
+cloudant.db=myDb
+~~~
+
+~~~ java
+@Autowired
+private Database db;
+
+public List<Greeting> getAllDocsAsGreetings() {
+    return List<Greeting> allDocs = db.getAllDocsRequestBuilder().includeDocs(true).build().getResponse().getDocsAs(Greeting.class);
+}
+~~~
+
+To provide custom connection data you can override the `com.cloudant.client.api.ClientBuilder` bean:
```

Comment:
Earlier:
>You **must** provide the following properties

Doesn't this override mean that you don't need to specify those properties? If so I think that should be made more clear here.